products/visualstudio.md

```diff
@@ -58,8 +58,8 @@ releases:
     lts: true
     eol: 2024-01-09
     releaseDate: 2022-05-10
-    latest: "17.2.18"
-    latestReleaseDate: 2023-08-08
+    latest: "17.2.19"
+    latestReleaseDate: 2023-09-09
```

Comment:
```suggestion
    latestReleaseDate: 2023-09-12
```

According to https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-v17.2#17.2.19.

Reply:
Thx good catch :+1: